llvm/lib/Target/AMDGPU/Utils/AMDGPUAsmUtils.cpp

```diff
@@ -669,5 +669,18 @@ const char* const IdSymbolic[] = {
 
 } // namespace VGPRIndexMode
 
+namespace UCVersion {
+
+ArrayRef<GFXVersion> getGFXVersions() {
+  static const GFXVersion Versions[] = {{"UC_VERSION_GFX7", 0},
```

Comment:
Why is this missing 6, 8, and 9?

Reply:
According to SP3, they don't support `s_version` and there are no `UC_VERSION_GFX*` codes for them.

Reply:
Can you add a comment for this

Reply:
Done.